qhub/template/stages/07-kubernetes-services/modules/kubernetes/services/conda-store/config/conda_store_config.py

```diff
@@ -129,6 +129,10 @@ async def authenticate(self, request):
 
```

Comment:
We will need to modify the new property that will be exposed in this PR https://github.com/Quansight/conda-store/pull/388. `c.CondaStore.environment_directory = "/home/conda/{namespace}/envs/{namespace}-{name}"`. This will ensure that the conda environment name has the namespace embeded within it.

Reply:
Bump the version to 0.4.12 as well